lib/windowMaid.js

```diff
@@ -0,0 +1,86 @@
+// Third Party Modules
+const _ = require('lodash')
+
+// Logging
+const debug = require('debug')
+const logger = debug('electronpdf:')
+
+/** How long a window can remain open before it is terminated, in milliseconds */
+const HUNG_WINDOW_THRESHOLD = process.env.ELECTRONPDF_WINDOW_LIFE_THRESHOLD || 1000 * 60 * 5 /* minutes */
+
+// Window Cache - Keep track of all windows created, and if any get stuck close
+// them
+const windowCache = {}
+
+class WindowMaid {
```

Comment:
I've been using Classes in this module.  Is there a real benefit to using an object literal instead of the class? Or you just think its a bit more concise?